ui/xournalpp.css

```diff
@@ -7,9 +7,23 @@
  *  See: https://wiki.gnome.org/action/show/Projects/GTK/Inspector
  */
 
-/*Style the Floating Toolbox!*/
+#pdfFloatingToolGrid {
+    padding: 7px;
+}
+
+#pdfFloatingToolGrid button {
```

Comment:
Make sure you handle the 'window.darkMode' case, too! (See lines 67-85)

Reply:
Oh, okay. In my opinion the floating toolbox should have same background color with the pdf and I couldn't find how to make pdf background black, so I didn't handle that case(I want to try to add the function to siwtch pdf light/dark mode on the fly and then add the `window.darkMode` for pdf floating toolbox then), thus you point out this it's okay to add dark mode for toolbox now.